app/date-select/controller.js

```diff
@@ -0,0 +1,34 @@
+const FormWizardController = require('../../common/controllers/form-wizard')
+const { dateRegex } = require('../../common/helpers/url')
+
+const DateRegExp = new RegExp('/' + dateRegex)
+
+class DateSelectController extends FormWizardController {
```

Comment:
I thought we weren't going to use the FormWizard for this control, as it's a single UI element, not a wizard collecting data?

Reply:
Ah! This is a separate form for entering the date that is being jumped to!